### ISSUE TITLE
Syria cropland labels

### ISSUE THREAD

**Issue by hannah-rae** [nasaharvest/cropharvest#122]
[Li et al., 2022](https://www.nature.com/articles/s43016-021-00432-4) provides 238,471 training samples for crop/non-crop in Syria over 2002-2019 via Google Drive: https://drive.google.com/drive/folders/1HLI3YAfCXcaccFJPQuhmgSlvfYuy4cBK

Description from the paper:
> We used a supervised classification approach to map annual productive cropland in Syria. We collected training data manually by visual interpretation using growing-season and non-growing-season Landsat images, time series of NDVI from Landsat and MODIS, and high-resolution images on Google Earth.
In the process of selecting training samples, we considered two critical factors to ensure the representativeness and accuracy of the training: (1) the training samples need to be selected from multiple years, covering the dry year (2007), the wet year (2002) and the year with moderate precipitation (2012) to ensure the diversity of the samples; (2) high-resolution images on Google Earth are adequate to ensure the accuracy of visual interpretation of various land-cover types. In the end, 238,471 training pixels covering the years 2002, 2007, 2012, 2017, 2018 and 2019 were selected on the Google Earth Engine platform.

**Comment by hannah-rae** [comment]
It looks like this dataset actually doesn't have lat/lon metadata. They are just numpy files with the covariates and labels. I'll ask Xiaopeng if they have the data with lat/lon that they are willing to publish.
